.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "airbnb",
+    "plugin:import/errors",
+    "plugin:import/warnings"
+  ],
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "plugins": [
+    "react",
+    "import"
+  ],
+  "rules": {
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "react/prefer-stateless-function": [0, { "ignorePureComponents": true }],
+    "import/no-unresolved": "off"
+  }
+}
```

Comment:
Let's use the config file here instead: https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json

If you are having issues with the linter free feel to discuss it on this PR. Also you can disable rules in the source code when it is reasonable using inline comments. See https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments